### PR TITLE
Issue 635 - leave handelbars comments as unformatted

### PR DESCRIPTION
**Pull request by donwalling**
This is a code suggestion for issue 635 to leave handelbars comments unformatted like html comments. I haven't figured out the test case heirarchy yet to understand where to put a test case. Suggestions would be appreciated.

### CODE REVIEW

**Comment by bitwiseman** [comment]
Haven't reviewed, but at least need tests.

**Comment by bitwiseman** [comment]
@donwalling - we're going to need tests for this before I can review it.

**Comment by donwalling** [comment]
Hi @bitwiseman - does https://github.com/donwalling/js-beautify/commit/2bcf4153748389cbca6dfd7f6787c3bdab26e841 suffice for tests? Basically re-using what is already there in a second variation. The challenge I ran into with other approaches is that mustache is stripping out the {{! }} comments when the script creates the beautify-html-tests.js file. 

BTW, I'm not sure why the beautify-html-tests.js file is checked in? it seems to be created by the testj Makefile target

File: js/lib/beautify-html.js [hunk 1]
```diff
@@ -536,6 +545,31 @@
                 return comment;
             };
 
+            this.get_handlebars_comment = function(start_pos) { //function to return handlebars comment content in its entirety
```
Comment (bitwiseman) [inline js/lib/beautify-html.js]: Any reason why this wouldn't be part of the general comment method above? it's mostly the same...

Comment (bitwiseman) [inline js/lib/beautify-html.js]: Shouldn't this be added to the comment handling method above instead of being made into a separate method?

Comment (donwalling) [inline js/lib/beautify-html.js]: indeed, it should have been - will submit another commit

**Comment by bitwiseman** [comment]
Your commit won't merge right now because I went in and made the handlebars tests part of the standard test template system.

**Comment by bitwiseman** [comment]
I rebased your changes to a single commit and then added the tests in the new format.   Thanks!
